GSA/digitalgov.gov
# Hotfix/rs convert jpg png

## Summary 

This hotfix addresses an issue where uploading JPG/JPEG images was causing errors and broken images on the website. The fix converts JPG/JPEG images to PNG format before further processing. Closes #7685.

### Preview

N/A

### Solution

- Added a new async function `convertJpgToPng` in `file-prep.js` to convert JPG/JPEG images to PNG format using the sharp library. 
- Modified the `fileTidy` function to call `convertJpgToPng` when a JPG/JPEG file is encountered. The original JPG file is deleted after conversion.
- Added error handling to log any errors that occur during the JPG to PNG conversion.
- Created a new `readme.md` file to provide an overview of the image and file upload processing functionality, usage instructions, configuration details, and dependencies.

### How To Test

1. Upload a JPG/JPEG image to the `content/uploads/_inbox` directory.
2. Run the `npx gulpupload` command.
3. Verify that the JPG/JPEG image is converted to PNG format in the `content/uploads/_working-images/to-process` directory.
4. Check the Images feed and Facebook Debugger to ensure the converted PNG image is displayed correctly without any errors.

---

### Dev Checklist
